Taudem5PCVS2010/ogrtest/ogrtest.vcxproj

```diff
@@ -0,0 +1,122 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This was useful in testing.  We may need to do some testing of this later on.  I think it is better to keep this in, rather than delete, even though this is not compiled as part of the release.  it should be documented that this is test/development code, but the name conveys this.